core/src/main/java/com/linecorp/armeria/internal/server/annotation/CompositeResponseConverterFunction.java

```diff
@@ -62,6 +63,12 @@ public HttpResponse convertResponse(ServiceRequestContext ctx,
         if (result instanceof HttpResponse) {
             return (HttpResponse) result;
         }
+        if (result instanceof HttpResult) {
```

Comment:
I'm not very familiar with the converter setup, so I'm not 100% sure if this is the only guaranteed entrypoint to the converter chain where we might receive an `HttpResult`.